Aron/tunnel refactor

amcconnellcloud opened the pull request:
This is the overall structure reformat to the Cloudflare Tunnel section of the dev docs. Part of a larger Cloudflare Tunnel dev docs overhaul.

ranbel commented on content/cloudflare-one/connections/connect-apps/deployment-guides/_index.md:
```diff
@@ -1,7 +1,7 @@
+weight: 5
 ---
 
 # Deploying `cloudflared`
```
FYI the directory listing shows all of the hidden links:

Ansible
AWS
Azure
Docker
GCP
Terraform
Kubernetes

ranbel commented:
@amcconnellcloud and @abelinkinbio
Made some changes to page headings andCLI page locations, and added public hostname/private network page intros. Check out the preview and lmk what you think:
https://4818798c.cloudflare-docs-7ou.pages.dev/cloudflare-one/connections/connect-apps/

amcconnellcloud commented:
Hi Ranbel,
Abe and I went over it and it looks great. I made a few minor changes. I added a link to the GitHub on the sidebar and the pages inside of routing to tunnel were being force reloaded since the redirected to themselves so I fixed that.
